backend/src/utils/users.js

```diff
@@ -0,0 +1,47 @@
+module.exports = {
+    validateUserObj: (req, res, next) => {
+        const user = req.body;
+        if (typeof user.username !== 'string' || user.username.length === 0) {
+            res.status(420).send('Username is not valid');
+        } else if (
```

Comment:
This logic for checking the phone number is now duplicated here and in `validateUserPhone`. Would be good if it was only in single function which is then used all around. This function is something that should be unit tested too.